lua/nvim-tree/actions/tree/modifiers/expand-all.lua

```diff
@@ -3,6 +3,7 @@ local renderer = require "nvim-tree.renderer"
 local Iterator = require "nvim-tree.iterators.node-iterator"
 local notify = require "nvim-tree.notify"
 local lib = require "nvim-tree.lib"
+local git = require "nvim-tree.git"
```

Comment:
Appears unused.

Reply:
it is used in the line https://github.com/nvim-tree/nvim-tree.lua/pull/2790/files#diff-b9bfb789bf93265f2e073fa63bcb9783b86a39e42c47b42b1e7193f87a03b489R33